pyupgrade/_plugins/lru_cache.py

```diff
@@ -39,3 +44,23 @@ def visit_Call(
             )
     ):
         yield ast_to_offset(node), _remove_call
+    elif (
+            state.settings.min_version >= (3, 9) and
+            not node.args and
+            len(node.keywords) == 1 and
+            node.keywords[0].arg == 'maxsize' and
+            isinstance(
+                node.keywords[0].value,
+                (ast.Constant, ast.NameConstant),
+            ) and
+            node.keywords[0].value.value is None and
+            is_name_attr(
+                node.func,
+                state.from_imports,
+                'functools',
+                ('lru_cache',),
+            ) and
+            'functools' not in state.from_imports and
+            'lru_cache' not in state.from_imports
```

Comment:
this part down here doesn't matter -- this should test for the `Attribute` directly and not use `is_name_attr` since we are not looking for from imports